fsys_test.go

```diff
@@ -0,0 +1,135 @@
+package skeletonkit_test
+
+import (
+	"embed"
+	"flag"
+	"io"
+	"path/filepath"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/gostaticanalysis/skeletonkit"
+	"github.com/tenntenn/golden"
+)
+
+type appInfo struct {
```

Comment:
Type can be defined into a function.
I think it is good that type definition near by using place.
It is similar with errMap.